chain/chain/src/chain.rs

```diff
@@ -3198,6 +3198,17 @@ impl Chain {
         prev_block_header: &BlockHeader,
         chunk: &ShardChunk,
     ) -> Result<(), Error> {
+        let protocol_version =
+            self.epoch_manager.get_epoch_protocol_version(block.header().epoch_id())?;
+
+        if checked_feature!(
+            "protocol_feature_relaxed_chunk_validation",
+            RelaxedChunkValidation,
+            protocol_version
+        ) {
+            return Ok(());
+        }
+
```

Comment:
note that this change is not required because it is not a part of chunk validation right now, but I think it still makes sense to have to avoid invalidating chunk which was already endorsed